persistence/persistence/src/main/java/one/microstream/persistence/types/PersistenceStorerCreatorDeactivateAble.java

```diff
@@ -0,0 +1,119 @@
+package one.microstream.persistence.types;
+
+/*-
+ * #%L
+ * MicroStream Persistence
+ * %%
+ * Copyright (C) 2019 - 2022 MicroStream Software
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import org.slf4j.Logger;
+
+import one.microstream.reference.ObjectSwizzling;
+import one.microstream.util.BufferSizeProviderIncremental;
+import one.microstream.util.logging.Logging;
+
+/**
+ * PeristenceStorer creator that creates {@link one.microstream.persistence.types.PersistenceStorerDeactivateAble#PersistenceStorerDeactivateAble PersistenceStorerDeactivateAble}
+ * instances.
+ * 
+ */
+public class PersistenceStorerCreatorDeactivateAble<D> implements PersistenceStorer.Creator<D>
```

Comment:
We should go with Deactivatable as suffix here.

Reply:
ok, done